man/standard-conf.xml

```diff
@@ -42,11 +42,12 @@
     <title>Configuration Directories and Precedence</title>
 
     <para>The default configuration is set during compilation, so configuration is only needed when it is
-    necessary to deviate from those defaults. Initially, the main configuration file in
-    <filename>/etc/systemd/</filename> contains commented out entries showing the defaults as a guide to the
-    administrator. Local overrides can be created by editing this file or by creating drop-ins, as described
-    below. Using drop-ins for local configuration is recommended over modifications to the main configuration
-    file.</para>
+    necessary to deviate from those defaults. The main configuration file is either in
+    <filename>/usr/lib/systemd</filename> or <filename>/etc/systemd/</filename> and contains commented out
```

Comment:
where is the "main configuration file in /usr/lib/systemd"? that's made up, just wrong.

Reply:
this does not use our usual rule that dir paths should be suffixed by /